### PR TITLE
Better Today Extension

### LINKED ISSUE #53: Today widget extension

### PR DESCRIPTION
**Pull request by mrkbryn**
Closes #53 

Created Today Extension with simplified LiteEatery and LiteEvent models to minimize cost of the extension. I had to reset to master and reimplement my work from the mark/today-widget branch because after rebasing with master on that branch, the branch got corrupted.

There's an issue with app groups that prevented me from fully implementing favorites (due to problems sharing favorites data from the main app to the extension via NSUserDefaults). However, the functionality for it is commented out in my changes on this branch so it can be easily integrated once this is fixed.

Currently, instead of listing favorites, the table view adds as many eateries (in order of fetching) and their hours descriptions until it reaches the max height for a table view in Notification Center.

### CODE REVIEW

**Comment by lucasderraugh** [comment]
This will have to wait until the DiningStack repo accepts the PR I have in to change the APIKey enum to be public. Until then this will just not compile.

File: Today Extension/LiteEatery.swift [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+class LiteEatery: NSObject {
+    static let dateKeyFormatter = makeKeyFormatter()
+    static let shortDateFormatter = makeShortDateFormatter()
+    
+    let id: Int
+    let nameShort: String
+    let slug: String
+    
+    private(set) var events: [String: [String: LiteEvent]] = [:]
+    private var _todaysEventsString: String? = nil
```
Comment (lucasderraugh) [inline Today Extension/LiteEatery.swift]: Wait why you have an underscore?

Comment (mrkbryn) [inline Today Extension/LiteEatery.swift]: It was from some of the code I took from Zielenski. I actually don't even use it so I'll take it out.

File: Today Extension/LiteEatery.swift [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    
+    init(json: JSON) {
+        id    = json[APIKey.Identifier.rawValue].intValue
```
Comment (lucasderraugh) [inline Today Extension/LiteEatery.swift]: Can we make `=` one space on either side.

**Pull request merged**